hashicorp/boundary-ui
# refactor: 💡 Updated Manage Dropdown to HDS in Targets Menu

Updated the manage dropdown present in the targets scope to reflect HDS design

:tickets: ✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14597

## Screenshots (if appropriate)
**Before**
<img width="2032" alt="Screenshot 2024-07-29 at 4 17 52 PM" src="https://github.com/user-attachments/assets/e89da655-6f23-49b0-8ccd-ebaf6ad407b8">
<img width="2030" alt="Screenshot 2024-07-29 at 4 18 24 PM" src="https://github.com/user-attachments/assets/3c78a63b-b52c-4d16-ad64-a222034ed79a">
**After**
<img width="2032" alt="Screenshot 2024-07-30 at 11 02 44 AM" src="https://github.com/user-attachments/assets/aa9e7f5e-7e02-46ef-9d42-4ded2b50cbc9">
<img width="2032" alt="Screenshot 2024-07-30 at 11 03 26 AM" src="https://github.com/user-attachments/assets/b46cf9c5-c887-40dc-94ff-e71830bbb9dc">


## How to Test
1. Setup yarn server
2. Select Targets menu and select a resource
3. Manage dropdown should appear on the right and reflect the new design.

## Checklist
- [] I have added JSON response output for API changes
- [] I have commented on my code, particularly in hard-to-understand areas
